code/modules/mob/living/brain/brain_cybernetic.dm

```diff
@@ -0,0 +1,62 @@
+/obj/item/organ/internal/brain/cybernetic
+	name = "cybernetic brain"
+	desc = "A mechanical brain found inside of androids. Not to be confused with a positronic brain."
+	organ_flags = ORGAN_ROBOTIC
+	failing_desc = "seems to be broken, and will not work without repairs."
+
+/obj/item/organ/internal/brain/cybernetic/brain_damage_examine()
+	. = ""
+	if(length(skillchips))
+		. += span_info("It has a skillchip embedded in it.")
```

Comment:
This skillchip line is copy/pasted between the two procs, you can leave it in normal brain examine.

Reply:
Actually, if you move this line out, the proc won't need to return a list - since it'll only ever return one string. So there ya go.